package.json

```diff
@@ -1,16 +1,23 @@
 {
   "name": "fweb3.xyz",
   "private": true,
+  "engines": {
+    "node": ">=16.14.0"
```

Comment:
I suspect this breaks the deployment on Vercel. They run on LTS versions so 14.x is the latest